### ISSUE TITLE
Attempt to invoke virtual method 'void com.horcrux.svg.CircleShadowNode.setR(java.lang.String)' on a null object reference

### ISSUE THREAD

**Issue by Almouro** [react-native-community/react-native-svg#812]
Hi,

Might be a duplicate of #811, this is a new issue for me after #806 was fixed for me with a forked version of RN.

I'm getting those two errors on `react-native-svg` `7.0.2`. They seem to affect all devices **exclusively** on Android 8.0. I unfortunately cannot reproduce them.

- Caused by: java.lang.NullPointerException (882 crashes aka about 1% of sessions I'd say)
  at com.horcrux.svg.RenderableViewManager$6.setR (RenderableViewManager.java:237)

- Caused by: java.lang.NullPointerException (126 crashes): 
  at com.horcrux.svg.RenderableViewManager.setFillRule (RenderableViewManager.java:717)

## Stacktrace

```
java.lang.RuntimeException: 
  at com.facebook.react.bridge.ReactContext.handleException (ReactContext.java:311)
  at com.facebook.react.uimanager.GuardedFrameCallback.doFrame (GuardedFrameCallback.java:31)
  at com.facebook.react.modules.core.ReactChoreographer$ReactChoreographerDispatcher.doFrame (ReactChoreographer.java:134)
  at com.facebook.react.modules.core.ChoreographerCompat$FrameCallback$1.doFrame (ChoreographerCompat.java:105)
  at android.view.Choreographer$CallbackRecord.run (Choreographer.java:909)
  at android.view.Choreographer.doCallbacks (Choreographer.java:723)
  at android.view.Choreographer.doFrame (Choreographer.java:655)
  at android.view.Choreographer$FrameDisplayEventReceiver.run (Choreographer.java:897)
  at android.os.Handler.handleCallback (Handler.java:789)
  at android.os.Handler.dispatchMessage (Handler.java:98)
  at android.os.Looper.loop (Looper.java:164)
  at android.app.ActivityThread.main (ActivityThread.java:6944)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:327)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1374)
Caused by: com.facebook.react.bridge.JSApplicationIllegalArgumentException: 
  at com.facebook.react.uimanager.ViewManagersPropertyCache$PropSetter.updateViewProp (ViewManagersPropertyCache.java:92)
  at com.facebook.react.uimanager.ViewManagerPropertyUpdater$FallbackViewManagerSetter.setProperty (ViewManagerPropertyUpdater.java:129)
  at com.facebook.react.uimanager.ViewManagerPropertyUpdater.updateProps (ViewManagerPropertyUpdater.java:48)
  at com.facebook.react.uimanager.ViewManager.updateProperties (ViewManager.java:32)
  at com.facebook.react.uimanager.NativeViewHierarchyManager.createView (NativeViewHierarchyManager.java:232)
  at com.facebook.react.uimanager.UIViewOperationQueue$CreateViewOperation.execute (UIViewOperationQueue.java:152)
  at com.facebook.react.uimanager.UIViewOperationQueue$DispatchUIFrameCallback.dispatchPendingNonBatchedOperations (UIViewOperationQueue.java:1012)
  at com.facebook.react.uimanager.UIViewOperationQueue$DispatchUIFrameCallback.doFrameGuarded (UIViewOperationQueue.java:983)
  at com.facebook.react.uimanager.GuardedFrameCallback.doFrame (GuardedFrameCallback.java:29)
Caused by: java.lang.reflect.InvocationTargetException: 
  at java.lang.reflect.Method.invoke (Native Method)
  at com.facebook.react.uimanager.ViewManagersPropertyCache$PropSetter.updateViewProp (ViewManagersPropertyCache.java:80)
Caused by: java.lang.NullPointerException: 
  at com.horcrux.svg.RenderableViewManager$6.setR (RenderableViewManager.java:237)
```

**Comment by msand** [comment]
Can you try with the latest commit from the plain-views branch?

**Comment by Almouro** [comment]
@msand unfortunately since I cannot reproduce locally, I'm not very keen on deploying a non stable version of this lib to them to validate the fix.
I'm downgrading to `6.5.2` for now.
Hopefully we'll find a way to reproduce with the version 7

**Comment by msand** [comment]
At least I think the plain-views branch should resolve specifically this kind of problems, as it's not using the shadow nodes to set any properties, they can't be null or cause exceptions. If the plain-views branch renders correctly for you locally, and all press handlers / gesture responders behave correctly, then I would say it's safe use the plain-views branch and publish a new version using that. At this point I don't know of any regressions, so essentially I'm just giving a bit of time in hopes of getting feedback on it, but could just as well cut a new release.

**Comment by msand** [comment]
@Almouro  Is this issue still occuring?

**Comment by msand** [comment]
I would assume this is fixed since removing the shadow nodes. Otherwise, we can probably introduce a null check or try to isolate the issue better. Please comment or reopen if you still observe exceptions with the latest version.
